corehq/apps/app_manager/static/app_manager/js/preview_app.js

```diff
@@ -149,6 +149,15 @@ hqDefine('app_manager/js/preview_app.js', function() {
         $togglePreviewBtn.click(_private.toggleAppPreview);
 
         var _resizeAppPreview = function () {
+
+            var $nav = $(layoutController.selector.navigation),
+                $alerts = $('.alert-maintenance');
+            var maxHeight = $appPreview.find('.preview-phone-container').outerHeight() + $nav.outerHeight() + 80;
```

Comment:
I see this is pre-existing, but the `80` is fragile, is it a height/padding/margin that could be measured from the page rather than hard coded?